Without this, the compiler complains about a prototype mismatch and the build fails.  Potentially send upstream

--- gcr-3.16.0/gcr/gcr-certificate-chain.h	2015-04-06 11:28:46.000000000 -0700
+++ gcr-3.16.0/gcr/gcr-certificate-chain.h	2015-05-27 15:23:54.276255958 -0700
@@ -90,14 +90,14 @@
 gboolean                  gcr_certificate_chain_build              (GcrCertificateChain *self,
                                                                     const gchar *purpose,
                                                                     const gchar *peer,
-                                                                    guint flags,
+                                                                    GcrCertificateChainFlags flags,
                                                                     GCancellable *cancellable,
                                                                     GError **error);
 
 void                      gcr_certificate_chain_build_async        (GcrCertificateChain *self,
                                                                     const gchar *purpose,
                                                                     const gchar *peer,
-                                                                    guint flags,
+                                                                    GcrCertificateChainFlags flags,
                                                                     GCancellable *cancellable,
                                                                     GAsyncReadyCallback callback,
                                                                     gpointer user_data);
